subt/cloudsim2osgar.py

```diff
@@ -66,13 +66,25 @@ def __init__(self):
         rospy.init_node('imu2osgar', log_level=rospy.DEBUG)
         rospy.loginfo("waiting for {}".format(imu_name))
         rospy.wait_for_message(imu_name, Imu)
+        # drone specific code
+        top_scan_name = '/'+robot_name+'/top_scan'
+        rospy.loginfo("waiting for {}".format(top_scan_name))
+        rospy.wait_for_message(top_scan_name, LaserScan)
+        bottom_scan_name = '/'+robot_name+'/bottom_scan'
+        rospy.loginfo("waiting for {}".format(bottom_scan_name))
+        rospy.wait_for_message(bottom_scan_name, LaserScan)
+        # end drone specific code
```

Comment:
If we merge this, it will stop working for all other robots because the messages will never come.

Reply:
Stupid me, you are absolutely right, sorry :( ... any idea how to reasonably do the ROS part? (on OSGAR I would read it from config ... hmm two launch files? Or extra param in `run_solution.bash`?

Reply:
maybe pass also ROBOT_TYPE (i.e. replace IS_X4 by names?)

Reply:
Yes. As I said elsewhere, it might be also time to switch from bash to python.

Reply:
solved by @zwn in #622